lib/msal-browser/src/network/FetchClient.ts

```diff
@@ -22,11 +23,16 @@ export class FetchClient implements INetworkModule {
             method: HTTP_REQUEST_TYPE.GET,
             headers: this.getFetchHeaders(options)
```

Comment:
We still want to wrap this in a try/catch, right?

Reply:
Done